src/main/kotlin/no/nav/syfo/brev/arbeidstaker/ArbeidstakerBrevApi.kt

```diff
@@ -63,8 +63,8 @@ fun Route.registerArbeidstakerBrevApi(
                 )
 
                 val hasAccessToBrev = motedeltakerArbeidstaker.personIdent == requestPersonIdent
-                if (hasAccessToBrev) {
-                    val pdf = pdfService.getPdf(brev.pdfId)
+                if (hasAccessToBrev && brev.pdfId != null) {
```

Comment:
Burde dialogmoteService.getArbeidstakerBrevFromUuid(brevUuid) bare burde returnere brev som har pdf i stedet? Samme gjelder for dialogmoteService.getNarmesteLederBrevFromUuid(brevUuid). Det er vel ikke aktuelt å hente ut noe annet arbeidstaker/arbeidsgiver APIet.

Reply:
Ja, men jeg er ganske sikker på at det er sånn det er: alt som rerurneres her skal ha pdf, men det vet ikke kompilatoren. Er det bedre å bare ha brev.pdfId!! i stedet?

Reply:
Ja, riktig for den henter bare opp ferdige referat og de andre brevene vil alltid ha pdf? Da må det vel nesten bli sånn, med mindre vi innfører en egen klasse for de uten pdf eller noe sånt 🤔

Reply:
Det stemmer 👍